test/functional/p2p_invalid_messages.py

```diff
@@ -39,19 +43,22 @@ def __repr__(self):
 class InvalidMessagesTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 1
-        self.setup_clean_chain = True
+        self.setup_clean_chain = False
```

Comment:
why is this changed?

Reply:
i thought it was faster but can revert that change while rebasing

Reply:
why would it be faster to copy the blocksdir as opposed to copy nothing?

Reply:
I (mis)understood(?) the discussions to say that using the cached chain was faster than spinning up a new one. Reverted.

Reply:
There is no new chain generated in this test:

```
$ git grep generate 'test/functional/p2p_invalid_messages.py'
$